src/main/java/com/google/codeu/data/Country.java

```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.data;
```

Comment:
Overall, I think the code looks pretty good. It might be easier to break your code up into multiple pull requests rather than doing it all in one next time. We might consider how a Country class might be similar/different to a Subchannel or subreddit class. Also, do we want users to be able to post directly in the Country class or have them find separate subreddits in the country page and then post on those respective channels instead? I think we should keep discussing with as a team how we'd like to handle cases like these.

Reply:
I do like having small PRs, but the problem here is that in creating a new datastore class (so that countries exist) I have to create an awful lot of files and change some things around. Most of the changes here need to all be implemented together or the code breaks.  I probably could have held off on the "messages can be posted to country pages" bit, but I hadn't thought of it and also it was a relatively minor change that may have been more confusing to make down the line. 

/country/[country]/[subchannel] might be a good layout to have - possibly we can use the country servlet to do checks that country exists, then have different functions for loading different subchannels. As for messages, we'd have to add a "channel" data field when storing and  processing them

I know that we pretty much decided we will not post directly to the Country page, but to me, changing message functionality from to-user to to-country was a nice half-step, where it's easy to build off of and change later, as people are actually building the subchannels they can change the messaging capability as needed.